components/Tabs2/src/MDCTabBarView.h

```diff
@@ -19,4 +19,13 @@
  */
 __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScrollView
 
+/** The set of items displayed in the Tab bar. */
+@property(nonnull, nonatomic, copy) NSArray<__kindof UITabBarItem *> *items;
+
+/** The currently-selected item in the Tab bar. */
+@property(nullable, nonatomic, strong) __kindof UITabBarItem *selectedItem;
+
+/** Selects one of the items with optional animation. */
+- (void)setSelectedItem:(__kindof UITabBarItem *)selectedItem animated:(BOOL)animated;
```

Comment:
Please remove all of the `__kindof` annotations, unless there is a strong technical reason for providing them (please explain it if so). See go/__kindof-scary for a discussion on this annotation, and https://stackoverflow.com/a/33799210 for more details on why this can lead to unexpected crashes.

tl;dr: This annotation hides legitimate warnings that come from down-casting an instance to a different type than what is specified by the generics annotations.

E.g.:

```objc
NSMutableArray<__kindof UIView *> *subviews = [[NSMutableArray alloc] init];

[subviews addObject:[[UIView alloc] init]]; // Works
[subviews addObject:[[UIImageView alloc] init]]; // Also works

subviews[0].image = [UIImage ...]; // Crash, no compile-time warning
```

Reply:
Let's remove it for now. I know the API docs specified using `__kindof` (to follow UIKit's more recent conventions), but we can always add it back in later.  Better to move fast and not debate things in a PR.

Reply:
SGTM